src/models/Application.py

```diff
@@ -112,3 +130,38 @@ def update_score(user_id, score):
             raise e
 
         return application
+
+    @staticmethod
+    def get_scored_applications():
+        """Returns all scored applications"""
+        return db.session.query(Application) \
+            .filter(Application.score != 0) \
+            .all()
+
+    @staticmethod
+    def standardize_scores():
+        """Updates the standardized scores for all applications"""
+        scored_applications = Application.get_scored_applications()
+
+        # gets all unique user ids
+        user_ids = {application.scoring_user_id for application in scored_applications}
+
+        # creates a dict to map between user_id and list of applications scored by that user
+        user_to_applications = {
+            user_id: list(
+                filter(lambda application: application.scoring_user_id == user_id,
+                       scored_applications))
+            for user_id in user_ids
+        }
+
+        # creates a dict to map between user_id and list of applications scored by that user with standardized_scores set
+        user_to_applications_with_standardized_scores = {
+            user_id: Application.set_standardized_scores(applications)
+            for user_id, applications in user_to_applications.items()
```

Comment:
this dict isn't used, again, if we do this as a single query and we can do it as one big transaction that would be more elegant